lib/krane/kubernetes_resource/custom_resource.rb

```diff
@@ -68,7 +68,7 @@ def validate_definition(*)
       @crd.validate_rollout_conditions
     rescue RolloutConditionsError => e
       @validation_errors << "The CRD that specifies this resource is using invalid rollout conditions. " \
-      "Kubernetes-deploy will not be able to continue until those rollout conditions are fixed.\n" \
+      "krane will not be able to continue until those rollout conditions are fixed.\n" \
```

Comment:
This is kinda a funny case--both versions of the command will now use the new name here. I think this is not a big enough deal to fix.